slack-connector/cmd/main.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"os"
+
+	"github.com/kyma-incubator/hack-showcase/slack-connector/internal/registration"
+	log "github.com/sirupsen/logrus"
+)
+
+func main() {
+	log.Info("Registration started.")
+
+	builder := registration.NewPayloadBuilder(registration.NewFileReader(), os.Getenv("SLACK_CONNECTOR_NAME"))
+	id, err := registration.NewApplicationRegistryClient(builder, 5, 10).RegisterService()
+	if err != nil {
+		log.Fatal("Fatal error: ", err.Error())
+	}
+	log.WithFields(log.Fields{
+		"id": id,
+	}).Info("Service registered.")
+
+	for true { // Temporary solution
```

Comment:
uhh?

Reply:
We considered changing Deployment to Job - this would fix the problem of finishing runtime of app and restarting pod.